Gems/Atom/Feature/Common/Code/Source/ImGui/ImGuiPass.cpp

```diff
@@ -140,23 +140,61 @@ namespace AZ
             m_drawData.push_back(drawData);
         }
 
-        int ImGuiPass::GetTickOrder()
+        ImGuiPass::TickHandlerFrameStart::TickHandlerFrameStart(ImGuiPass& imGuiPass)
+            : m_imGuiPass(imGuiPass)
+        {
+            TickBus::Handler::BusConnect();
```

Comment:
I don't see where the TickBus disconnect is called (for both handlers)

Reply:
All EBus handlers automatically disconnect in the base class destructor.